css/forms.css

```diff
@@ -0,0 +1,234 @@
+.range-wrap {
+  display: flex;
+  align-items: center;
+  position: relative;
+  width: 100%;
+  height: 5.25em;
+  font: 1em/1 arial, sans-serif;
+}
+
+[type=range] {
+  flex: 1;
+  margin: 0;
+  padding: 0;
+  min-height: 1.5em;
+  background: transparent;
+  font: inherit;
+}
+[type=range], [type=range]::-webkit-slider-thumb {
+  -webkit-appearance: none;
+}
+[type=range]::-webkit-slider-runnable-track {
+  box-sizing: border-box;
+  border: none;
+  width: 12.5em;
+  height: 0.25em;
+  background: #ccc;
+}
+[type=range]::-moz-range-track {
+  box-sizing: border-box;
+  border: none;
+  width: 12.5em;
+  height: 0.25em;
+  background: #ccc;
+}
+[type=range]::-ms-track {
+  box-sizing: border-box;
+  border: none;
+  width: 12.5em;
+  height: 0.25em;
+  background: #ccc;
+}
+[type=range]::-webkit-slider-thumb {
+  margin-top: -0.625em;
+  box-sizing: border-box;
+  border: none;
+  width: 1.5em;
+  height: 1.5em;
+  border-radius: 50%;
+  background: var(--indigo);
+}
+[type=range]::-moz-range-thumb {
+  box-sizing: border-box;
+  border: none;
+  width: 1.5em;
+  height: 1.5em;
+  border-radius: 50%;
+  background: #f90;
```

Comment:
Can make use of color variables, [here](https://github.com/venkateshwarans/AI-Carte/pull/2/files#diff-900257eeb8c177ccdbe96e3e5f8958acbccb7f21cdb0b9a1b421634f9bc1d85fR66) too.